test/e2e/suite_test.go

```diff
@@ -174,6 +213,9 @@ var _ = SynchronizedAfterSuite(func() {
 }, func() {
 	// After all ParallelNodes.
 
+	By("Dumping logs from the bootstrap cluster")
+	dumpBootstrapClusterLogs(bootstrapClusterProxy)
```

Comment:
This By and dumpBootstrapClusterLogs call looks like leftover debugging tools that don't really make sense in this testing (`By`) context.

Reply:
I've removed this